packages/insomnia/src/utils/importers/importers/postman.ts

```diff
@@ -136,9 +136,11 @@ export class ImportPostman {
       url: this.importUrl(request.url),
       parameters: parameters,
       method: request.method || 'GET',
-      headers: headers.map(({ key, value }) => ({
+      headers: headers.map(({ key, value, disabled, description }) => ({
         name: key,
         value,
+        disabled: disabled || false,
+        description,
```

Comment:
```suggestion
        ...(typeof disabled !== 'undefined' ? { disabled } : {}),
        ...(typeof description !== 'undefined' ? { description } : {}),
```
In the insomnia RequestHeader type these fields are optional so make them conditionally inserted would best fit that type.